markup.go

```diff
@@ -47,36 +47,32 @@ type Event struct {
 
 // MarkupOrComponentOrHTML represents one of:
 //
-//  Markup
+//  MarkupList
 //  Component
 //  *HTML
+//  ChildList
 //
 // If the underlying value is not one of these types, the code handling the
 // value is expected to panic.
 type MarkupOrComponentOrHTML interface{}
 
 func apply(m MarkupOrComponentOrHTML, h *HTML) {
-	if m == nil {
-		return
-	}
 	switch m := m.(type) {
-	case Markup:
-		m.Apply(h)
-	case Component:
-		h.children = append(h.children, m)
-	case *HTML:
+	case MarkupList:
```

Comment:
This case would just become `Applyer`

Reply:
See https://github.com/gopherjs/vecty/pull/134#discussion_r135370011